drawing_1.css

```diff
@@ -0,0 +1,26 @@
+.container{
+	width: 250px;
+	height: 100px;	
```

Comment:
Taking out the width and height of the container `.container` did not affect the drawing.